mitx/lms/static/sass/base/_reset.scss

```diff
@@ -41,13 +41,15 @@ body.view-in-course .content-wrapper,
 }
 
 #main {
-    position: relative;
-    z-index: 3;
 
     .view-profile & {
         max-width: inherit;
         padding: 0;
     }
+
+    .staff-modal {
+        margin-left: -559px !important;
```

Comment:
Please make it `margin-left: -40% !important;`

Reply:
Updated